.travis.yml

```diff
@@ -13,16 +13,16 @@ stages:
   install: false
   script: false
 
-.check_moban: &check_moban
-  <<: *disable_global
-  python: 3.6
-  stage: moban
-  install: pip install moban
-  script: .ci/check_moban.sh
+# .check_moban: &check_moban
```

Comment:
i think you should not comment this

Reply:
I have uncommented it. This will .cause the moban check of travis to fail. After https://github.com/coala/coala/pull/6043 is merged, the moban will pass.